core/src/com/unciv/ui/pickerscreens/PickerScreen.kt

```diff
@@ -11,43 +11,40 @@ open class PickerScreen : CameraStageBaseScreen() {
 
     internal var closeButton: TextButton = Constants.close.toTextButton()
     protected var descriptionLabel: Label
-    protected var rightSideGroup = VerticalGroup()
+    private var rightSideGroup = VerticalGroup()
     protected var rightSideButton: TextButton
-    private var screenSplit = 0.85f
 
     /**
      * The table displaying the choices from which to pick (usually).
      * Also the element which most of the screen realestate is devoted to displaying.
      */
     protected var topTable: Table
     var bottomTable:Table = Table()
-    internal var splitPane: SplitPane
     protected var scrollPane: ScrollPane
 
     init {
         bottomTable.add(closeButton).pad(10f)
 
         descriptionLabel = "".toLabel()
         descriptionLabel.wrap = true
-        val labelScroll = ScrollPane(descriptionLabel,skin)
+        val labelScroll = ScrollPane(descriptionLabel)
         bottomTable.add(labelScroll).pad(5f).fill().expand()
 
         rightSideButton = "".toTextButton()
         rightSideButton.disable()
         rightSideGroup.addActor(rightSideButton)
 
         bottomTable.add(rightSideGroup).pad(10f).right()
-        bottomTable.height = stage.height * (1 - screenSplit)
 
         topTable = Table()
         scrollPane = ScrollPane(topTable)
 
-        scrollPane.setSize(stage.width, stage.height * screenSplit)
-
-        splitPane = SplitPane(scrollPane, bottomTable, true, skin)
-        splitPane.splitAmount = screenSplit
-        splitPane.setFillParent(true)
-        stage.addActor(splitPane)
+        val pickerScreenTable = Table()
```

Comment:
Why do you feel that the absolute split is better than the percentage based split?

Reply:
it's very good for Portrait screen

Reply:
Interesting